packages/authentication/src/__tests__/fixtures/user-profile-spike/user.repository.ts

```diff
@@ -0,0 +1,48 @@
+import {MyUser} from './user';
+
+export class MyUserRepository {
+  constructor(readonly list: {[key: string]: MyUser}) {}
+  find(name: string): MyUser | undefined {
+    const found = Object.keys(this.list).find(k => this.list[k].name === name);
+    return found ? this.list[found] : undefined;
+  }
+}
+
+export const SAMPLE_USER_MIM_SET = {
```

Comment:
> SAMPLE_USER_MIM_SET 

What's a MIM set?  Did you mean MIN as in minimum?

Reply:
oh good catch on the typo